static/app/views/settings/dynamicSampling/dynamicSampling.tsx

```diff
@@ -0,0 +1,125 @@
+import {TargetSampleRateField} from 'sentry/views/settings/dynamicSampling/targetSampleRateField';
+import {useAccess} from 'sentry/views/settings/projectMetrics/access';
+
+const {useFormState, FormProvider} = dynamicSamplingForm;
+
+export function DynamicSampling() {
+  const api = useApi();
+  const organization = useOrganization();
+  const {hasAccess} = useAccess({access: ['org:write']});
+
+  const formState = useFormState({
+    targetSampleRate: ((organization.targetSampleRate ?? 1) * 100)?.toString(),
+    samplingMode: 'auto' as const,
+  });
+
+  const modeField = formState.fields.samplingMode;
+  const endpoint = `/organizations/${organization.slug}/`;
+
+  const {mutate: updateOrganization, isPending} = useMutation({
+    mutationFn: () => {
+      const {fields} = formState;
+      return api.requestPromise(endpoint, {
+        method: 'PUT',
+        data: {
+          targetSampleRate: Number(fields.targetSampleRate.value) / 100,
+        },
+      });
+    },
+    onSuccess: () => {
+      addSuccessMessage(t('Changes applied.'));
+      formState.save();
+    },
+    onError: () => {
+      addErrorMessage(t('Unable to save changes. Please try again.'));
+    },
+  });
+
+  const handleSubmit = () => {
+    updateOrganization();
+  };
+
+  const handleReset = () => {
+    formState.reset();
+  };
+
+  return (
+    <FormProvider formState={formState}>
+      <form onSubmit={event => event.preventDefault()}>
+        <Panel>
+          <PanelHeader>{t('Automatic Sampling')}</PanelHeader>
+          <PanelBody>
+            <FieldGroup
+              label={t('Sampling Mode')}
+              help={t('Changes the level of detail and configuring sample rates.')}
+            >
+              {t('Automatic Balancing')}
+            </FieldGroup>
+            {/* TODO(aknaus): move into separate component when we make it interactive */}
+            <FieldGroup
+              disabled
+              label={t('Switch Mode')}
+              help={t(
+                'Take control over the individual sample rates in your projects. This disables automatic adjustments.'
+              )}
+            >
+              <Fragment>
+                <Confirm disabled>
+                  <Button
+                    title={t('This feature is not yet available.')}
+                    css={css`
+                      width: max-content;
+                    `}
+                  >
+                    {modeField.value === 'auto'
+                      ? t('Switch to Manual')
+                      : t('Switch to Auto')}
+                  </Button>
+                </Confirm>
+              </Fragment>
+            </FieldGroup>
+            {modeField.value === 'auto' ? <TargetSampleRateField /> : null}
+          </PanelBody>
+        </Panel>
+        <FormActions>
+          <Button disabled={!formState.hasChanged || isPending} onClick={handleReset}>
+            {t('Reset')}
+          </Button>
```

Comment:
should we add a tooltip informing the users why the button is disabled? It is pretty self-explanatory to me but imo it would be nice to have.